doc/source/runastrodriz.rst

```diff
@@ -61,3 +61,430 @@ for each input image.
 The '-n' option allows the user to specify the number of cores to be used in
 running AstroDrizzle.
 
+
+.. _runastrodriz-description:
+
+Description
+===========
+A lot of effort goes into trying to determine the WCS solution which provides
+closest agreement with the GAIA astrometry frame.  Combining the 
+images successfully requires the most accurate alignment possible between all 
+the input exposures taking into account the best available distortion model
+for the input exposures.  Being able to compare the combined image to other exposures
+taken of the same field at other times, or even with other telescopes, requires
+that the WCS be defined to come as close to the GAIA frame as possible.  The 
+processing done by `runastrodriz` attempts to not only apply the most current 
+distortion model, but also the best available pre-computed GAIA-based WCS 
+solutions while proceeding to determine it's own solution to the available GAIA
+reference stars for the field-of-view.  It then performs numerous checks to see 
+which of these solutions results in the most accurately aligned images to each 
```

Comment:
The sentence starting with "It then performs..." is a bit clumsy at the end. Consider rewording.

Reply:
Revised (heavily) in e5791399.